apps/web/src/pages/ApplicantDashboardPage/components/QualifiedRecruitments/QualifiedRecruitmentStatus.tsx

```diff
@@ -0,0 +1,194 @@
+            data-h2-height="base(1em)"
+            data-h2-width="base(1em)"
+          />
+        </span>
+        <p>
+          {intl.formatMessage({
+            defaultMessage:
+              "This recruitment has expired and it is no longer available for hiring opportunities.",
+            id: "sfa4iJ",
+            description:
+              "Expired recruitment status message on qualified recruitment card.",
+          })}
+        </p>
+      </div>
+    );
+  }
+
+  // Recruitment card is active and not suspended status.
+  if (!suspendedAt) {
+    return (
+      <div data-h2-display="base(flex)" data-h2-align-items="base(center)">
+        <span data-h2-padding="base(0, x.5, 0, 0)" data-h2-color="base(green)">
+          <CheckCircleIcon
+            data-h2-height="base(1em)"
+            data-h2-width="base(1em)"
+          />
+        </span>
+        <p>
+          {intl.formatMessage({
+            defaultMessage:
+              "You are actively appearing in talent results for this recruitment.",
+            id: "TDikDt",
+            description:
+              "Not suspended recruitment status message on qualified recruitment card.",
+          })}
+          {/* Dialog that allows user to suspend the qualified recruitment (application). Sets suspendedAt date to the current time. */}
+          <QualifiedRecruitmentDialog
+            id={id}
+            isSuspended={!!suspendedAt}
+            closeButtonLabel={intl.formatMessage({
+              defaultMessage: "Yes, remove me from search results",
+              id: "wgRXhR",
+              description:
+                "Dialog close button on change search results status from qualified recruitment section.",
+            })}
+            openDialogLabel={intl.formatMessage({
+              defaultMessage: "Remove me",
+              id: "2g1mfB",
+              description:
+                "Open dialog text on change search results status from qualified recruitment section.",
+            })}
+            primaryBodyText={intl.formatMessage({
+              defaultMessage:
+                "You are currently appearing in talent search results for the “IT-01 Helpdesk support technician” pool.",
+              id: "dPoE9z",
+              description:
+                "Dialog main body on change search results status from qualified recruitment section.",
+            })}
+            secondaryBodyText={intl.formatMessage({
+              defaultMessage:
+                "If you’ve recently been placed or simply no longer want to be considered for opportunities related to this role, you can remove yourself from the list of candidates available for hire. This will NOT remove you from the recruitment itself and you can always re-enable your availability if you change your mind.",
+              id: "EWB2LC",
+              description:
+                "Dialog main body on change search results status from qualified recruitment section.",
+            })}
```

Comment:
Ugh, sorry I did not notice this earlier. Can we avoid using all capitals since some ATs will treat it like an acronym and read "N-O-T" instead of "not"? Instead, we should use emphasis (either `strong` or `em`). The problem being, our current `strong` actually does not emphasize text to avoid misuse 😖 

You can either:

1. Make a custom one here or;
2. Add one to our richTextElements.tsx

Option 2 is preferred but, causes problems since we are already using `strong` to apply font-weight so naming could be tricky. Maybe add a custom one for now, I believe we do have an issue to sort out bold vs emphasized text 🤔

Reply:
Alright I added in a new `emphasize` wrapper to our richFormattedText.tsx file. It uses the `strong` tag along with bolding the text. When we get that issue you mentioned we can rename/fix the current setup. [f1201f6](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/f1201f68d51eab2e903fe74e37667ad1af4c945c)